### DISCUSSION TITLE
clash调成tun模式的时候显示无网络连接，clash for windows也一样，同样配置公司电脑就可以

### DISCUSSION THREAD

**Discussion by sansan-mei** [zzzgydi/clash-verge#409] [Q&A]
![image](https://user-images.githubusercontent.com/68650762/218304251-6787aaac-6589-44c4-a7c8-e3a4af5197e0.png)
![image](https://user-images.githubusercontent.com/68650762/218304306-df4b45e8-32f0-47ef-8f2c-e140ab3828e5.png)

**Comment by fatsnk** [comment]
可以参考下https://github.com/zzzgydi/clash-verge/issues/410
显示无法连接Internet并不影响上网，如果确实是无法正常使用，可以试试用管理员权限运行程序
